index.js

```diff
@@ -36,9 +36,9 @@
 //. That is, it takes an argument of type `Number` and returns a value of
 //. type `Number`.
 //.
-//. [`R.map`][R.map] has type `(a -> b) -> [a] -> [b]`. That is, it takes
+//. [`S.map`](#map) has type `(a -> b) -> [a] -> [b]`. That is, it takes
```

Comment:
This is the only reference to the `R.map` link identifier, so we can delete the relevant line from the bottom of the file.